app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find_by id: params[:id]
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+
+    if @user.save
+      flash[:success] = t ".signup_success"
+      redirect_to @user
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit :name, :email, :password, :password_confirmation
```

Comment:
```suggestion
    params.require(:user).permit User::USER_PARAMS
```
ở model user tạo constant USER_PARAMS = %i(name email password password_confirmation).freeze nha.